### DISCUSSION TITLE
Media Plugin (Android && iOS)

### DISCUSSION THREAD

**Discussion by avalontm** [AvaloniaUI/Avalonia#12543] [General]
Hello!, make an adaptation of a xamarin plugin for avalonia

can Take Photo, Take video, Pick Photo,  Pick video

<a href="https://ibb.co/w4gQkhh"><img src="https://i.ibb.co/1TQXgGG/screen1.png" alt="screen1" border="0"></a>

<a href="https://ibb.co/9TNynsB"><img src="https://i.ibb.co/T8kPcmJ/screen3.png" alt="screen3" border="0"></a>

<a href="https://ibb.co/9rjq1JX"><img src="https://i.ibb.co/gRCDfG1/screen2.png" alt="screen2" border="0"></a>

**Comment by timunie** [comment]
looks great 👍 Maybe you want to share a link to that project? Also when it's finished, consider to add it to Awsome-Avalonia https://github.com/AvaloniaCommunity/awesome-avalonia

**Reply by avalontm** [reply]
okas, later
